data/mods/Xedra_Evolved/recipes/inventor/armor.json

```diff
@@ -0,0 +1,160 @@
+[
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "helmet_inventor",
+    "using": [ [ "soldering_standard", 20 ], [ "welding_standard", 10 ] ],
+    "qualities": [ { "id": "SAW_M", "level": 2 }, { "id": "SAW_M_FINE", "level": 1 }, { "id": "GRIND", "level": 1 } ],
+    "proficiencies": [
+      { "proficiency": "prof_elec_soldering", "time_multiplier": 3, "skill_penalty": -100 },
```

Comment:
So what does negative 100 do when the doc describes it at `"skill_penalty": 1.5 // The effective skill penalty when crafting without this proficiency. Defaults to 1.0. Multiple proficiencies will add to this value.`?

Reply:
It makes the recipe to apply proficiency slowdown, but prevent the recipe from fail result, and i thought about it as a feature of all  inventor's recipes

Reply:
Makes sense.